script/build

```diff
@@ -65,7 +65,7 @@ cmake $@ "${SRC_DIR}"
 
 MAX_THREADS_TO_BUILD_WITH=8
 NUM_THREADS_TO_BUILD_WITH=$(getconf _NPROCESSORS_ONLN)
-if [ ${NUM_THREADS_TO_BUILD_WITH} -gt ${MAX_THREADS_TO_BUILD_WITH} ]
+if [ "${NUM_THREADS_TO_BUILD_WITH}" -gt ${MAX_THREADS_TO_BUILD_WITH} ]
```

Comment:
This works for comparing strings; while 'gt' is about comparing integers

Reply:
These are integers in this case. So maybe neither of them needs to be in quotes? Perhaps I will revert this change.

Reply:
Fixed